plugins/analytics/public/virustotal_api.py

```diff
@@ -128,9 +144,12 @@ def analyze(observable, results):
                 new_hash = Hash.get_or_create(value=v)
                 new_hash.tag(observable.get_tags())
                 links.update(
-                    new_hash.active_link_to(observable, k, 'virustotal_query'))
+                    new_hash.active_link_to(observable, k,
+                                            'virustotal_query'))
+
+            result['scan_date'] = json_result['scan_date']
 
         result['source'] = 'virustotal_query'

```

Comment:
You should probably add this `source` key in the case `json_result['response_code'] != 1`